src/test/java/org/fcrepo/transform/http/FedoraTransformTest.java

```diff
@@ -101,6 +97,7 @@ public void setUp() {
 
     @Test
     @SuppressWarnings("unchecked")
+    @Ignore("this is a silly test")
```

Comment:
I would suggest using this test, or removing this class.
As it stands right now, the class does nothing.